imap_processing/idex/idex_l1a.py

```diff
@@ -312,9 +312,9 @@ def _set_impact_time(self, packet: space_packet_parser.parser.Packet) -> None:
               testing.
         """
         # Number of seconds since epoch (nominally the launch time)
-        seconds_since_launch = packet.data["SHCOARSE"].derived_value
+        seconds_since_launch = packet.user_data["SHCOARSE"]
```

Comment:
Do we need `.raw_value` or `.derived_value` here? I don't fully understand when the `_Parameter` type is returned...

Reply:
The `_Parameter` is the derived value. But, SHCOARSE is never derived it is just a counter, so it is the same as the raw value too.